pkgs/applications/video/freetube/default.nix

```diff
@@ -1,51 +1,109 @@
-{ stdenv, lib, fetchurl, appimageTools, makeWrapper, electron_22 }:
+{ stdenv
+, lib
+, fetchurl
+, undmg
+, appimageTools
+, makeWrapper
+, electron_22
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "freetube does not support system: ${system}";
```

Comment:
This is unnecessary, as the evaluation will already fail if the system isn't in `meta.platforms`.